app/src/main/res/layout/fragment_timeline.xml

```diff
@@ -1,13 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v4.widget.SwipeRefreshLayout
+<RelativeLayout
```

Comment:
Why `RelativeLayout` here?
I would either:
1. Make it a `FrameLayout`
2. Make it a `ConstraintLayout`
3. Make the `CoordinatorLayout` a parent and put `SwipeRefreshLayout` inside alongside that include (probably the best choice)

Reply:
I guess because I don't have strong feelings about it - I can adapt it to whichever of the above seems best.